MapboxCoreNavigation/RouteController.swift

```diff
@@ -635,6 +646,10 @@ extension RouteController: CLLocationManagerDelegate {
         updateRouteStepProgress(for: location)
         updateRouteLegProgress(for: location)
         
+        if routeProgress.currentLegProgress.currentStep.containsTunnel {
```

Comment:
Move this if into `detectRouteStepInTunnel(for:)` and return early if it's true.